apps/pkeyparam.c

```diff
@@ -125,13 +124,8 @@ int pkeyparam_main(int argc, char **argv)
              * Note: at least for RSA keys if this function returns
              * -1, there will be no error reasons.
              */
-            BIO_printf(out, "Parameters are invalid\n");
-
-            while ((err = ERR_peek_error()) != 0) {
-                BIO_printf(out, "Detailed error: %s\n",
-                           ERR_reason_error_string(err));
-                ERR_get_error(); /* remove err from error stack */
-            }
+            BIO_printf(bio_err, "Parameters are invalid\n");
```

Comment:
... and line 121

Reply:
I do not understand what you do not like here and in the two other places?

Reply:
@levitte is this PR okay to merge now also in your view?

Reply:
The print statements that you refer here and below are positive ones, such as
```
BIO_printf(out, "Parameters are valid\n");
```
so I see little point in replacing `out` by `bio_err` for them.